ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -102,6 +109,19 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
         }
     }
 
+    private fun isTokenizationKey(): Boolean {
+        var isTokenizationKey = false
```

Comment:
Would it make more sense for this to return a `Boolean?` and set it to `null` initially? Edge case, but in the event that there is an error returning auth, we are setting `isTokenizationKey` to `false`, which would then allow the shopper insights API request to proceed without error.

Reply:
I think there is a problem to proceed with tokenizationkey and we want to send error if it is.